src/MatrixClientPeg.ts

```diff
@@ -218,7 +218,7 @@ class MatrixClientPegClass implements IMatrixClientPeg {
         opts.pendingEventOrdering = PendingEventOrdering.Detached;
         opts.lazyLoadMembers = true;
         opts.clientWellKnownPollPeriod = 2 * 60 * 60; // 2 hours
-        opts.experimentalThreadSupport = SettingsStore.getValue("feature_threadstable");
+        opts.experimentalThreadSupport = true;
```

Comment:
Should this be renamed or deleted?

Reply:
This will need to be renamed, but will require a breaking change on the js-sdk.
This won't be removed as we still need to support client that do not wish to support threads.

Reply:
OK, is that rename tracked somewhere so we don't forget?